config/environment.js

```diff
@@ -26,7 +26,7 @@ module.exports = function (environment) {
   if (environment === 'development') {
     // ENV.APP.LOG_RESOLVER = true;
     // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
+    ENV.APP.LOG_TRANSITIONS = true;
```

Comment:
I normally leave this commented out, but it's helpful to utilize when weird things are happening with routing.